frontend/common/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import Link from "../Link";
+
+export interface BreadcrumbsProps {
+  links: { title: string; href: string }[];
+}
+
+const Breadcrumbs: React.FunctionComponent<BreadcrumbsProps> = ({ links }) => {
+  // taking in the array of links as defined above, create an array of Link components
+  const arrayLinks = links.map((link, index) => (
+    // eslint-disable-next-line react/no-array-index-key
+    <Link href={link.href} title={link.title} key={index}>
+      {link.title}
+    </Link>
```

Comment:
As an alternative to inserting ">" between links later, you could also include them here like so:
```suggestion
    <React.Fragment key={index}>
      {/* Include a ">" before every link but the first */}
      {index > 0 && <span data-h2-padding="b(right-left, xs)">&gt;</span>}
      {/* Some breadcrumb sections may not actually be a link */}
      {link.href ? (
        <Link href={link.href} title={link.title}>
          {link.title}
        </Link>
      ) : (
        <p>{link.title}</p>
      )}
    </React.Fragment>
```

Reply:
React.Fragments? Oh, something to try to wrap my head around 🤯